### ISSUE TITLE
Integrate plotting and tabular printing functions to Tellurium

### ISSUE THREAD

**Issue by kirichoi** [sys-bio/telluriumUtilities#1]
Plotting will be integrated into [tellurium.visualization](https://github.com/sys-bio/tellurium/tree/master/tellurium/visualization) submodule and printing will be integrated into [tellurium.utils](https://github.com/sys-bio/tellurium/tree/master/tellurium/utils) submodule.

**Comment by 0u812** [comment]
This isn't intended to be a complete replacement for the plotting code already in Tellurium, correct?

**Comment by kirichoi** [comment]
You are correct. These are specialized plotting functions for heatmaps, etc. which should work on top of plotting engine we already have.
